docs/examples/opentelemetry-example-app/setup.cfg

```diff
@@ -41,11 +41,11 @@ packages = find_namespace:
 zip_safe = False
 include_package_data = True
 install_requires = 
-    opentelemetry-api == 0.16.dev0
-    opentelemetry-sdk == 0.16.dev0
-    opentelemetry-instrumentation-requests == 0.15.b0
-    opentelemetry-instrumentation-flask == 0.15.b0
-    opentelemetry-instrumentation-grpc == 0.15.b0
+    opentelemetry-api == 0.16b0
+    opentelemetry-sdk == 0.16b0
+    opentelemetry-instrumentation-requests == 0.16b0
```

Comment:
Oh actually the `lint` command fails on this too, hm, not too sure how the releases did this in the past when these packages hadn't been released at this version yet?